modules/renderer/features.js

```diff
@@ -210,7 +199,7 @@ export function rendererFeatures(context) {
 
         for (var i = 0; i < strings.length; i++) {
             var s = strings[i];
-            if (past_futures[s] || past_futures[tags[s]]) { return true; }
+            if (osmLifecyclePrefixes[s] || osmLifecyclePrefixes[tags[s]]) return true;
```

Comment:
Does it make sense that `intermittent:*` would be filtered out by the past/future filter? (Also `not:*`, discussed above.)

Reply:
not sure what the right answer is for `intermittent:*` is, but bear in mind that [`intermittent:*` is only used once](https://taginfo.osm.org/search?q=intermittent:) in the world (cf. other prefixes like [`disused:*`](https://taginfo.osm.org/search?q=disused:) are used >200,000 times). It seems like `intermittent=yes` is the more common tag.

Reply:
Looks like the `intermittent` entry originated from 494e247ad1ec458d0d2bed6102dd5deb90eb11aa. It was probably intended to support `intermittent=yes`, not `intermittent:*=*`.

Reply:
Thanks for finding that. I guess it would be logical for the past/future filter to match all the lifecycle prefixes except `intermittent`